### PR TITLE
Many problems

### PR DESCRIPTION
**Pull request by Mistwolf0423**
# Change
* Combine a lot of features into a single branch

### CODE REVIEW

File: alphabetical_ordering.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            d.append(char)
+        else:
+            c.append(char)
+    else:
+        if char.isupper():
+            a.append(char)
+        else:
+            b.append(char)
+
+print(''.join(sorted(b))+''.join(sorted(a))+''.join(sorted(c))+''.join(sorted(d)))
```
Comment (GermanCalle) [inline alphabetical_ordering.py:19]: Add empty line at the end

File: arrays.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+n = int(input())
+
+coor = []
+for i in range(x+1):
+    for j in range(y+1):
+        for k in range(z+1):
+            if i+j+k == n:
+                continue
+            coor.append([i,j,k])                
+print(coor)
```
Comment (GermanCalle) [inline arrays.py:13]: Same as arthimetical ordering

File: arrays_2.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+def arrays(arr):
+    arr1 = numpy.array(arr, dtype = float)
+    arr2 = numpy.array(arr[::-1], dtype = float)
+    return arr2
+
+arr = input().strip().split(' ')
+result = arrays(arr)
+print(result)
```
Comment (GermanCalle) [inline arrays_2.py:8]: Same problem. Add empty line at the end

File: average.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    n = int(input())
+    student_marks = {}
+    for _ in range(n):
+        name, *line = input().split()
+        scores = list(map(float, line))
+        student_marks[name] = scores
+    query_name = input()
+    na = student_marks[query_name]
+    dt = sum(na)/len(na)
+    print("%.2f" % dt)
```
Comment (GermanCalle) [inline average.py:11]: Oh my. You need an autoformater to remember that last line.

File: consecutive_values.py [hunk 5]
```diff
@@ -0,0 +1,5 @@
+if __name__ == '__main__':
+    n = int(input())
+    
+for i in range(1,n+1):
+    print(i, end="")
```
Comment (GermanCalle) [inline consecutive_values.py:5]: Yeap. You need it.

File: funcion_is_leaps.py [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+        leap = True
+    elif year % 100 == 0:
+        leap = False
+    elif year % 4 ==0 :
+        leap = True
+    return leap
+
+
+year = int(input())
+print(is_leap(year))
```
Comment (GermanCalle) [inline funcion_is_leaps.py:13]: Maybe can try with black package

File: list.py [hunk 7]
```diff
@@ -0,0 +1,10 @@
+n = int(input())
+arr = map(int, input().split())
+
+def remove_n(list, n):
+    arr_2 = [num for num in list if num != n]
+    return arr_2
+
+arr = list(arr)
+arr_2 = remove_n(arr, max(arr))
+print(max(arr_2))
```
Comment (GermanCalle) [inline list.py:10]: ;c

File: list_commands.py [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+        if s[0]=='append':
+            ls.append(int(s[1]))
+        if s[0]=='sort':
+            ls.sort()
+        if s[0]=='pop':
+            ls.pop()
+        if s[0]=='reverse':
+            ls.reverse()     
+        if s[0]=='print':
+            print(ls)
```
Comment (GermanCalle) [inline list_commands.py:19]: :/

**Review by GermanCalle — CHANGES_REQUESTED** [review]
Add empty line at the end of your scripts.

File: alphabetical_ordering.py (hunk 1 above)
Comment (Mistwolf0423) [reply alphabetical_ordering.py]: Done

**Review by GermanCalle — APPROVED** [review]

**Pull request merged**